### DISCUSSION TITLE
Is a user-space error to be expected when operation is aborted?

### DISCUSSION THREAD

**Discussion by kubijo** [enisdenjo/graphql-ws#505] [Q&A] [answered]
In the example below, should I get an error with an "aborted" reason, or is it expected that the library code "swallows" it?
My experiments tell me that the latter is true, but I haven't been able to deduce whether that is a correct and expected behavior.

If it's relevant, I am using [`cross-fetch`](https://www.npmjs.com/package/cross-fetch) as a `fetch` polyfill.

```typescript
const abortCtrl = new AbortController();
abortCtrl.abort();

const { data, error } = await client.query(
    /* query */ gql`…`,
    /*  vars */ {},
    /*   ctx */ { fetchOptions: { signal: abortCtrl.signal } },
).toPromise();
```

**Comment by enisdenjo** [comment, marked as answer]
I think your question arrived at a wrong place. 😅 

This is a WebSockets library, it doesn't use the Fetch API whatsoever.

**Reply by kubijo** [reply]
Hmm... right... it is more likely, though, that I just got a little confused with that additional information... 

Or is the whole question of aborting an operation (subscription in case of this library) out of context here and I should ask in urql?

**Reply by enisdenjo** [reply]
What I am trying to say is that this library does not use `fetch` (hence also no `AbortController`) at all, and you're asking about `fetch`.

I think you might want to post this question to the relevant library.
